delete copies of the lexicon

CrazyBoy49z opened the pull request:
### What does it do?
delete copies of the lexicon key setting_passwordless_activated and setting_passwordless_activated_desc

### Why is it needed?
https://github.com/modxcms/revolution/blob/3.x/core/lexicon/en/setting.inc.php#L845-L846

https://github.com/modxcms/revolution/blob/3.x/core/lexicon/en/setting.inc.php#L851-L852

CrazyBoy49z committed:
delete copies of the lexicon

delete copies of the lexicon key setting_passwordless_activated and setting_passwordless_activated_desc

JoshuaLuckers commented:
Good catch! 

There are other duplicates related to the passwordless login:
- https://github.com/modxcms/revolution/blob/3.x/core/lexicon/en/setting.inc.php#L848-L849
- https://github.com/modxcms/revolution/blob/3.x/core/lexicon/en/setting.inc.php#L854-L855

CrazyBoy49z committed:
delete copies lexicon

delete copies lexicon key setting_passwordless_expiration and setting_passwordless_expiration_desc

Ibochkarev reviewed (approved):
Good find.

CrazyBoy49z commented:
> Good find.

When translating into Ukrainian, the same key came across
